GuessGame.py

```diff
@@ -2,6 +2,7 @@
 from time import sleep
 import keyboard

```

Comment:
You may consider adding a `requirements.txt` to the repository and include the package `keyboard` in it.